app/assets/stylesheets/application/themes.scss

```diff
@@ -989,6 +989,49 @@ $lever-black-80: #342f2e;
     }
   }
 
+  // EPUB Reader
+  #reader {
+
+    .cozy-control .cozy-h1 {
+      font-size: 1.5em;
+    }
+
+    .button {
+      @include epub-button-background-color($lever-brand-color,$lever-accent-color,$lever-white);
+    }
+
+    .button--lg {
+      @include epub-button-background-color($lever-brand-color,$lever-accent-color,$lever-white);
+    }
+
+    .button--sm {
+      @include epub-button-background-color($lever-brand-color,$lever-accent-color,$lever-white);
+    }
+
+    i[class^='icon-chevron-'] {
```

Comment:
Just double checking:  [class^='icon-chevron-'] is correct syntax.

Reply:
yes